### ISSUE TITLE
Error with 0.9.0

### ISSUE THREAD

**Issue by DarkIsDude** [xkjyeah/vue-google-maps#384]
Hi,

Thanks for your amazing library first of all! I try to update to 0.9.0 but I have the following error:

```
Module build failed: ReferenceError: Unknown plugin "transform-inline-environment-variables" specified in "/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/vue2-google-maps/.babelrc" at 1, attempted to resolve relative to "/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/vue2-google-maps"
    at /Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (<anonymous>)
    at Function.normalisePlugins (/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)
    at OptionManager.init (/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at transpile (/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-loader/lib/index.js:50:20)
    at Object.module.exports (/Users/doudou/Workspaces/ESAQUE/front.git/src/node_modules/babel-loader/lib/index.js:173:20)
```

In my `main.js`:

```
import * as VueGoogleMaps from 'vue2-google-maps'
Vue.use(VueGoogleMaps, { load: { key: process.env.GOOGLE.MAPS } })
```

**Comment by xkjyeah** [comment]
Hm... thanks for reporting the error.

Could you try:
```sh
$ echo '{}' > node_modules/vue2-google-maps/dist/.babelrc
```

Does this fix the error?

**Comment by DarkIsDude** [comment]
Yes, this fix the error 👏 .

Here my .babelrc (default with vue-cli, don't touch it since 4 months).

```
{
  "presets": [
    ["env", {
      "modules": false
    }],
    "stage-2"
  ],
  "plugins": ["transform-runtime"],
  "env": {
    "test": {
      "presets": ["env", "stage-2"],
      "plugins": ["transform-es2015-modules-commonjs", "dynamic-import-node"]
    }
  }
}
```

**Comment by xkjyeah** [comment]
Forgot an empty .babelrc, so the project-wide babelrc was applied to files in `dist`.

Should be fixed in latest version (0.9.2)
